tests/color/test_grayscale.py

```diff
@@ -0,0 +1,40 @@
+import jax
+import keras_core as keras
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+from kornia.color.gray import grayscale_from_rgb
+
+
+# TODO: make this a fixture
+def genrate_image_data(channel_first: bool = False):
```

Comment:
I had imagined something like this to be added into `testing/`

Reply:
i mean, this testing https://github.com/kornia/kornia/tree/keras-core/testing

which should live codes that are just utilities for tests, but isn't tests or fixtures itself

Reply:
Like the BaseTester and assert close, it would not necessarily have to be part of the kornia package